pkgs/build-support/build-fhsenv-bubblewrap/default.nix

```diff
@@ -18,6 +18,7 @@
 , passthru ? {}
 , extraPreBwrapCmds ? ""
 , extraBwrapArgs ? []
+, multiArch ? false # Whether to include 32bit packages
```

Comment:
If we're lifting this up to the wrapper, could you remove the default from the builder? That way it's clear where it's declared.